.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: node_js
 node_js:
 - '8'
-- '9'
```

Comment:
The new version of `ava` doesn't seem to like Node 9. I don't think we really care about Node 9 so this seemed fine to me.